openapi.yaml

```diff
@@ -369,6 +359,23 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/JsonError'
+              examples:
+                runerror: 
+                  value: 
+                    error: "Error Retrieving Runs"
+                  summary: An error occured when trying to retireve the runs
+                pageerror:
+                  value: 
+                    error: "Error retrieving page"
+                  summary: An error occured when trying to return a results page
+                instanterror:
+                  value:
+                    error: "Error parsing Instant"
+                  summary: An error occurred trying to interpret the date-time parameters in to and from fields
+                runiderror:
+                  value:
+                    error: "Error retrieving run cdb-a4ddb7fd1dab8d6025e4f3894010d20d"
+                  summary: An error occured when trying to retrieve a specific run  using a runId
   /ras/run/{runid}/runlog:
     get:
       operationId: getRasRunLog
```

Comment:
Generally it's best practice to number each error message (see the CLI for an example) eg: GAL8001E: An internal server error occurred when trying to retrieve a specific run using the runid 'xxx'.
Wondering if we can start numbering them. Perhaps will be easier if we create an Error class... we could have an enumeration of the numbers used.

Reply:
That's so when the error message is translated, the number stays the same. I know we are not translating (yet) but the principle is sound.